type.proto

```diff
@@ -14,7 +14,8 @@ enum LabelType {
   LABEL_CYCLIST = 3;
   LABEL_MISC = 4;
   LABEL_GROUND = 5;
-  LABEL_MAX = 6;
+  LABEL_GATE = 6;
+  LABEL_MAX = 7;
```

Comment:
Do we need this value? Protobuf automatically generates the max enum value like ArraySize so it's redundant in my opinion.